src/main/java/com/stuypulse/robot/subsystems/Climber.java

```diff
@@ -5,24 +5,155 @@
 
 package com.stuypulse.robot.subsystems;
 
+import java.io.Console;
+
+import com.revrobotics.CANSparkMax;
+import com.revrobotics.RelativeEncoder;
+import com.revrobotics.CANSparkMaxLowLevel.MotorType;
+import com.stuypulse.robot.Constants;
+
+import edu.wpi.first.wpilibj.DigitalInput;
+import edu.wpi.first.wpilibj.Encoder;
+import edu.wpi.first.wpilibj.Solenoid;
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
 import edu.wpi.first.wpilibj2.command.SubsystemBase;
 
 /*-
  * Climbs at end of match
  *
  * Contains:
- *      - TBD
+ *      - Change tilt of climber
+ *      - Move climber via motor
+ *      - Different tilt angles
+ *      - Encoder + Solenoid used for stopping
  *
- * @author [TBD]
+ * @author independence106(Jason Zhou)
+ * @author Ca7Ac1(Ayan Chowdhury)
+ * @author marcjiang7(Marc Jiang)
+ * @author ambers7(Amber Shen)
+ * @author Souloutz(Howard Kong)
+ * @author jiayuyan0501(Jiayu Yan)
+ * @author ijiang05(Ian Jiang)
+ * @author TraceyLin (Tracey Lin)
+ * @author annazheng14(Anna Zheng)
+ * @author lonelydot(Raymond Zhang)
+ * @author andylin2004 (Andy Lin)
  */
 public class Climber extends SubsystemBase {
 
+    private Solenoid solenoidLong;
+    private Solenoid solenoidShort;
```

Comment:
this is just a naming thing, but you can probably remove solenoid from the name cause it's in the class. the stopper has a good name.

Reply:
also from my understanding, these solenoids are for different climber arm angles. if you want my name suggestion try smth like alpha, beta? whatever you name it put a comment explaining